Mathmagician.Tests/UnitTest1.cs

```diff
@@ -6,9 +6,29 @@ namespace Mathmagician.Tests
     [TestClass]
```

Comment:
At some point you guys can get rid of this file completely if it doesn't have any of the tests for your app in it.